tasks/lib/uglify.js

```diff
@@ -94,7 +94,7 @@ exports.init = function(grunt) {
           minifyOptions.mangle.properties.reserved = [];
         }
         if (options.reserveDOMProperties) {
-          require('uglify-js/tools/domprops').forEach(function(name) {
+          require('uglify-es/tools/domprops').forEach(function(name) {
```

Comment:
BTW, how about moving this require to the top along with the rest? It seems better from a maintenance point of view.